README.md

```diff
@@ -1,20 +1,62 @@
 # Ising_programs
 
-May 23, 2020
-Doug Barrick
+A suite of python programs to analyze repeat-protein unfolding data with a 1D Ising model. 
 
-A suite of python programs to analyze repeat-protein unfolding data with a 1D Ising model.
+Homopolymer 1D Ising Fit           |  Heteropolymer 1D Ising Fit
+:-------------------------:|:-------------------------:
+![homopolymer_fit](homopolymer_ff_construct.png)  |  ![heteropolymer_fit](heteropolymer_ff_construct.png)
 
-This repository contains four folders.  The two main folders contain scripts to run 1D Ising analysis.
+
+This repository contains four main folders. The two main folders (```heteropolymer_fit``` and ```homopolymer_fit```) contain scripts to run 1D Ising analysis.
 One is for "capped homopolymeric" NRC-type repeats, and the other is for capped heteropolymeric NRXC-type repeats.
 Both perform nonlinear least-squares fits generated plots, and perform and statistical analysis using boostrap resampling.
 
-In addition there is a folder with some additional scripts to convert and merge data files of different types.
+In addition there is a folder (```data_conversion```) with some additional scripts to convert and merge data files of different types.
 
-Finally, there is a folder for calculation of partial correlation coefficients from bootstrapped thermodynamic parameters.
+Finally, there is a folder for calculation of partial correlation coefficients from bootstrapped thermodynamic parameters (```partial_correlation```).
 
-All folders contain data files on which the scripts can be run.  All programs were written to run in python 2.7 except 
-for the partial correlation program, which requires python 3.7.
+All folders contain data files which contain protein folding data on which the scripts can be run.  All programs were written to run in python 3.8.
```

Comment:
@barricklab-at-jhu - comment about python 3.8 or 3.7. 

I think it's customary to say what you wrote the code in. The environment.yml file includes this python dist. I have also seen people just colloquially write python 2 or python 3. When a user follows the instructions for env creation, conda may do all of this automagically when creating the environment. Let's test it together.